### PR TITLE
Judges endpoints

### PR DESCRIPTION
**Pull request by laurengreenway**
### New endpoints include:
- get judges by ID
- get judges in episode
- get judges in season

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,328 @@ An API to retrive information about _RuPaul's Drag Race_ Queens, Seasons, Episod
 
 ## Documentation
 
+<<<<<<< HEAD
```
Comment (GarrettLevine) [inline README.md]: looks like there's a BIG merge conflict here - we can just deleted it all. If you look at the README in master, it's like one line pointing to the readme docs.

File: lib/endpoints/judges/get-episode-judges.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+function getEpisodeJudges(req, res) {
+  return Episode.findById(req.params.id, {
+    include: [
+      {
+        model: Judge
+      }
+    ]
+  })
+    .then(cs => {
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-episode-judges.js]: I think we took `cs` from other file, and it stood for `challenges`, but let's make this `judges` or `js` or something liek that?

File: lib/endpoints/judges/get-judge-by-id.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+} = require('utils')
+
+function getJudgeById(req, res) {
+  const { id } = req.params;
+
+  return Judge.findById(id, {
+      include: [
+        {
+          model: Episode,
+          // through: {
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-judge-by-id.js]: if we're not using this code, let's remove it :D

File: lib/endpoints/judges/get-judge-by-id.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          // }
+         },
+      ],
+    })
+    .then(judge => {
+      if (!judge) return Promise.reject(eh.noJudgeWithId(id));
+      res.json(formatJudge(judge));
+    })
+    .catch(err => {
+      console.log({ err });
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-judge-by-id.js]: remove stray console log :ok_man:

File: lib/endpoints/judges/get-judges.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+} = require(`utils`);
+
+function getJudges(req, res) {
+  return Judge.findAll({
+    limit: req.query.limit || 20,
+    offset: req.query.offset || 0,
+    order: [['id', 'ASC']]
+  })
+  .then(judges => {
+    res.json(judges);
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-judges.js]: let's attach `.status(200)` to this response.

File: lib/endpoints/judges/get-season-judges.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    .then(episodes => {
+      return Promise.all(episodes.map(episode => episode.getJudges()));
+    })
+    .then(judges => {
+      let flatArray = [];
+      judges.forEach(eJudges => {
+        flatArray = [...flatArray, ...eJudges];
+      });
+      const unique = _.uniqBy(flatArray, 'id')
+      if (!judges.length)
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-season-judges.js]: I think we meant to move this to line 17, and return if there were no responses from the DB

File: lib/endpoints/judges/get-season-judges.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        return Promise.reject(eh.noJudgeWithId(req.params.id));
+      const formattedJudge = unique.map(j => {
+        return formatJudge(j.dataValues);
+      });
+      console.log('form at', formattedJudge)
```
Comment (GarrettLevine) [inline lib/endpoints/judges/get-season-judges.js]: stray console log!

File: lib/endpoints/judges/index.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+e.getJudges = require(`./get-judges`);
+e.getJudgeById = require(`./get-judge-by-id`);
+e.getEpisodeJudges = require(`./get-episode-judges`);
+e.getSeasonJudges = require(`./get-season-judges`);
+e.create = require(`./create`);
+
+module.exports = e;
+
+// TODO: create the following endpoints:
```
Comment (GarrettLevine) [inline lib/endpoints/judges/index.js]: let's remove these comments

**Comment by laurengreenway** [comment]
@GarrettLevine 🏁 🏁 🏁 🏁 🏁

File: README.md [hunk 9]
```diff
@@ -4,4 +4,326 @@ An API to retrive information about _RuPaul's Drag Race_ Queens, Seasons, Episod
...
+GET `http://www.nokeynoshade.party/api/queens/1/episodes`
+
+_example response:_
+```JS
+  {
+    [
+      {
+       "id": 1,
+       "title": "Drag on A Dime",
+       "episodeInSeason": 1,
```
Comment (GarrettLevine) [inline README.md]: heyyyy I am soooorry - but we actually don't need any of this! just the bit that points to the full documentation 🙈

**Pull request merged**